### PR TITLE
In progress

### PR DESCRIPTION
**Pull request by Georgeoak03**
pulling from inprogress to feature

### CODE REVIEW

**Comment by Georgeoak03** [comment]
pulling from inprogress to feature

File: portfolio.css [hunk 1]
```diff
@@ -277,8 +287,15 @@ footer {
 		border: none;
 	}
 
-	.contact-links {
+	#contact {
+		padding: 15px 100px;
+	}
+	/*.contact-links {
```
Comment (Harry-Johnston) [inline portfolio.css:293]: Remove this block of commented out code

File: portfolio.html [hunk 2]
```diff
@@ -30,6 +33,7 @@ <h1 class="portfolio-title">Portfolio.</h1>
 				<img class="dog1" src="images/dog1.jpg" alt="Doggos">
 				<h2 class="project-title1">PROJECT 1.</h2>
 				<p class="project-main-text1">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.</p>
+				<a href="#" class="github-link-portfolio"><i class="fa-brands fa-github-square"></i></a>
 			</div>
```
Comment (Harry-Johnston) [inline portfolio.html:37]: this /div is indented to the wrong line

File: portfolio.html [hunk 3]
```diff
@@ -38,8 +42,9 @@ <h2 class="project-title2">PROJECT 2.</h2>
...
+					<h2 class="project-title3">PROJECT 3.</h2>
+					<p class="project-main-text3">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.</p>
+				</div>
```
Comment (Harry-Johnston) [inline portfolio.html:47]: this /div is indented wrong i think

**Review by Harry-Johnston — CHANGES_REQUESTED** [review]

**Review by SlothSan — APPROVED** [review]
Approved this was a rename on your .html

**Review by SlothSan — APPROVED** [review]
Looks good, i like the animations.

**Comment by SlothSan** [comment]
Ready to merge my dude, good job.

**Pull request merged**